README.rst

```diff
@@ -0,0 +1,227 @@
+FunFlow
+=======
+
+.. highlight:: haskell
+.. default-role:: code
+
+.. image:: https://circleci.com/gh/tweag/funflow.svg?style=svg
+    :target: https://circleci.com/gh/tweag/funflow
+
+FunFlow is a library and tools to compose and run computational workflows. 
```

Comment:
This is not super clear to me. I think it'd be helpful to explain a little about why you'd want to use it, or what benefits it'll bring - WIIFM, basically.